src/services/rename.ts

```diff
@@ -38,9 +38,17 @@ namespace ts.Rename {
                     return undefined;
                 }
 
-                const displayName = stripQuotes(getDeclaredName(typeChecker, symbol, node));
                 const kind = SymbolDisplay.getSymbolKind(typeChecker, symbol, node);
-                return kind ? getRenameInfoSuccess(displayName, typeChecker.getFullyQualifiedName(symbol), kind, SymbolDisplay.getSymbolModifiers(symbol), node, sourceFile) : undefined;
+                if (!kind) {

```

Comment:
This change could use a test.

Reply:
We don't have a good way to test that it shows up in the right place in the UI, but I can try to add a lower-level test of the rename info.

Reply:
Just to clarify, what is the expected change to rename behavior due to this PR?

Reply:
Different strings are displayed in VS dialogs.  No impact on renaming.

Reply:
There must be some function that returns a different value than before in some situation?

Reply:
Yes, of course.  And I believe I've found a fourslash helper for testing it.  I just meant that the same user actions would result in the same result (i.e. renaming) after the change.